relationships/synthesis/APM-APPLICATION-to-REDIS.yml

```diff
@@ -0,0 +1,44 @@
+relationships:
+  - name: apmServiceCallsInfraElastiCacheRedisCluster
+    version: "1"
+    origins:
+      - APM Metrics
+    conditions:
+      - attribute: metricName
+        regex: "^Datastore/instance/Redis/[^/]*\\.cfg\\.[^/]*\\.cache\\.amazonaws\\.com/[0-9]+"
```

Comment:
I don't think we need two different rules just because of this minor change in the regex.
We don't care about this specific detail, the lookup is going to be the same (segments 4 and 5) against the `REDIS` category.

Reply:
BTW could you share a query containing this metric name so we can review that the regex is correct?

Reply:
@javimb Please find the query link here -> https://staging.onenr.io/0qQappbKPj1 
we were getting a different tagNames from the cluster entites, so we have separated the node and cluster

Reply:
Thanks for sharing. As you can see in [this query](https://staging.onenr.io/0EjOeeX98w6), there is no need to have two rules. `^Datastore/instance/Redis/[^/]*\\.cache\\.amazonaws\\.com/[0-9]+` matches for both clusters and nodes.